rate_counter/rate/tests.py

```diff
@@ -1,3 +1,55 @@
 from django.test import TestCase
-
+import utils
+from .models import TableRow
 # Create your tests here.
+
+
+class UtilsModuleTest(TestCase):
+    def test_no_table_rows(self):
+        """
+        without table rows, there is shouldn't be any result
+        """
+        table_rows = TableRow.objects.all()
```

Comment:
Ok, for now, in future we can switch to smth like [FactoryBoy](http://factoryboy.readthedocs.io/en/latest/introduction.html) for testing. to mock Models